webapp-django/crashstats/settings/base.py

```diff
@@ -536,6 +536,9 @@ def path(*dirs):
 # Credentials for being able to make an S3 connection
 AWS_ACCESS_KEY = config('AWS_ACCESS_KEY', None)
 AWS_SECRET_ACCESS_KEY = config('AWS_SECRET_ACCESS_KEY', None)
+AWS_HOST = config('AWS_HOST', None)
+AWS_PORT = config('AWS_PORT', 0, cast=int)
+AWS_SECURE = config('AWS_SECURE', True, cast=bool)
```

Comment:
These three get added to settings, but the code only uses them iff `AWS_HOST` and `AWS_PORT` have truthy values.